### PR TITLE
react-react1-week2

### PR DESCRIPTION
**Pull request by SvitlanaBurlaka**

### CODE REVIEW

File: react/react1/week2/my-app/src/components/TodoList.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            const newTodo = {
+                id: prevTodos[prevTodos.length - 1].id + 1,
+                description: "Random text",
+                status: false,
+            };
+            return [...prevTodos, newTodo];
+        });
+    };
+
+    function TaskDescription(props) {
```
Comment (dsamourkasidis) [inline react/react1/week2/my-app/src/components/TodoList.js]: Better to create a separate file for this component, even if it's only this small

File: react/react1/week2/my-app/src/components/TodoList.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        return (
+            <span>
+                {props.description}
+            </span>
+        )
+    };
+
+    const handleOnChange = (id) => {
```
Comment (dsamourkasidis) [inline react/react1/week2/my-app/src/components/TodoList.js]: Maybe use a better name here? handleOnChange is a bit vague, for instance, onTodoChecked

File: react/react1/week2/my-app/src/components/TimeCounter.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+import { useState, useEffect } from "react";
+
+export const TimeCounter = () => {
+    const [seconds, setSecondsCount] = useState(0);
+
+    useEffect(() => {
+        const interval = setInterval(() => {
+            setSecondsCount(seconds => seconds + 1)
+        }, 1000);
+        return () => clearInterval(interval);
```
Comment (dsamourkasidis) [inline react/react1/week2/my-app/src/components/TimeCounter.js:11]: good job cleaning up :)

**Review by dsamourkasidis — COMMENTED** [review]
Excellent work 💯 
Just two very very minor comments :)

File: react/react1/week2/my-app/src/components/TodoList.js (hunk 1 above)
Comment (SvitlanaBurlaka) [reply react/react1/week2/my-app/src/components/TodoList.js]: Thank you for review!

**Pull request merged**